### DISCUSSION TITLE
What is the correct way to monitor the same docker/host as ssm?

### DISCUSSION THREAD

**Discussion by Scott8586** [SquirrelCorporation/SquirrelServersManager#224] [Q&A]
I have several services running under docker on the same host on which I run ssm inside docker.  What is the correct way to set that up as a client?
No services show up when I simply install ssm server/master. I tried installing the agent on the same host - but I don't see any docker services (docker test passed when agent was installed).  The host I am running this under is a a PVE LXC for docker, docker version 27.1.2, docker compose version v2.29.2.  I've tried installing the agent as 127.0.01, and as the local external IP.

Docker services are showing up correctly from other hosts where I have installed the agent.
Thanks for any help.

**Comment by SquirrelDeveloper** [comment]
Hi @Scott8586 
SSM does not monitor docker containers automatically on its own host for the moment without some conf.
You must add a device and specify the private IP (so **not** localhost or 127.0.0.1) of the SSM host so it can work.
I will change that in the future
